doc/sailfish-webview-qmozcontext.cpp

```diff
@@ -0,0 +1,380 @@
+/****************************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+#include "sailfish-webview-qmozcontext.h"
+#include "webengine.h"
+
+// Dummy immplementation for documentation purposes only
+
+/*!
+    \class QMozContext
+    \brief Web engine context implementation.
+    \inmodule SailfishWebView
+
+    \c QMozContext is inherited by \l WebEngine to provide methods for
+    interacting with the underlying EmbedLiteApp and gecko engine.
+
+    Developers should access the methods through \l WebEngine rather than
+    instantiating \c QMozContext directly.
+*/
+
+/*!
+    \brief Registers the WebEngine for receiving notifications on a topic.
+
+    The gecko and EmbedLite components use notifications to transfer data and
```

Comment:
EmbedLite components are fine to be mentioned as those are part of low-level integration mechanisms.

Reply:
Thanks: left as-is.